spark/functions.py

```diff
@@ -1,2 +1,24 @@
 # setup()
-# draw()
\ No newline at end of file
+# draw()
+
+from ipycanvas import *
+
+canvas = Canvas(width = 400, height = 400)
```

Comment:
As a hard-coded canvas instance, would this make problems? Is it possible to get the `core_obj.canvas` instance instead?